mltsp/Flask/flask_app.py

```diff
@@ -1979,11 +1979,15 @@ def featurize_proc(
     # subprocess that is separate from main app:
     before_request()
     try:
-        results_str = featurize.featurize(
-            headerfile_path, zipfile_path, features_to_use=features_to_use,
-            featureset_id=featureset_key, is_test=is_test,
-            already_featurized=already_featurized,
-            custom_script_path=custom_script_path)
+        first_N = 5 if is_test else None
```

Comment:
Perhaps this should be hard-coded in `cfg` instead of here in `flask_app` so that as many of our hard-coded parameters are centralized there as possible.